packages/mui-material-next/src/Button/Button.tsx

```diff
@@ -416,6 +418,7 @@ const Button = React.forwardRef(function Button<
     centerRipple = false,
     children,
     className,
+    classes: classesProp,
```

Comment:
Out of curiosity, does the Material You component still accept `classes` prop?

Reply:
I left the API to be close with the current Material UI components. There are other adventages for the `classes` prop, like allowing state classes to be passed, like `disabled`, `active` etc. Also, we come to the understanding that it's a nice shortcut, as we've seen when working with the MUI Base classes (that do not support this API).